src/extension.ts

```diff
@@ -41,7 +42,15 @@ export async function activate(ctx: vscode.ExtensionContext) {
   ctx.subscriptions.push(
     vscode.debug.registerDebugConfigurationProvider(
       "cppdbg",
-      new DebugConfigurationProvider(buildDir),
+      new DebugConfigurationProviderCppdbg(buildDir),
+      vscode.DebugConfigurationProviderTriggerKind.Dynamic,
+    ),
+  );
+
+  ctx.subscriptions.push(
+    vscode.debug.registerDebugConfigurationProvider(
+      "lldb",
```

Comment:
Can you expose these strings as static public constants on the DebugConfigurationProvider implementations?

Reply:
Should be fixed by f17751a.